admin-manual/customization/authentication.rst

```diff
@@ -381,6 +381,391 @@ see:
      whose only initial permissions in AtoM will be those of the Authenticated
      :term:`user group`. No password will be stored in AtoM with this method, 
      since the LDAP server handles authentication. This is the default behavior 
-     in AtoM when LDAP is enabled and no user accounts are present. 
+     in AtoM when LDAP is enabled and no user accounts are present.
+
+.. _oidc-enabling:
+
+Enabling OIDC authentication
+============================
+
+AtoM can be configured to authenticate users using OpenID Connect (OIDC) instead
+of local authentication. When this is enabled, users attempting to log in will be
+redirected to an external identity and access management (IAM) server for
+authentication. After successfully authenticating, a user account will be created
+if one does not exist already. Logging out from AtoM will then log the user out
+from both AtoM and the IAM server.
+
+Enabling OIDC authentication requires manually editing a few config files.
+For more information on how to do this, see :ref:`customization-config-files`
+
+.. NOTE::
+
+   Proper operation of the OIDC Plugin requires configuring AtoM with an external
+   cache. See :ref:`oidc-cache-configuration` for more information.
+
+Enable the plugin
+-----------------
+
+Create an empty file named ``activate-oidc-plugin`` in the root of the AtoM
+installation to enable the OIDC plugin. Permissions and ownership of this file
+should match the other folders in the AtoM folder.
+
+Add the plugin name to the ``$plugins`` array in ``config/ProjectConfiguration.class.php``
+
+.. image:: images/enable-oidc-plugin.*
+   :align: center
+   :width: 90%
+   :alt: Plugin name ``arOidcPlugin`` added in the ``$plugins`` array.
```

Comment:
Skip the instructions in section 1.1.2 in the OIDC doc. The is replaced by section 1.1.1 (adding the empty file named 'activate-oidc-plugin').